Makefile

```diff
@@ -3,9 +3,9 @@ SOURCE_DATE_EPOCH ?= $(shell date +%s)
 BUILD_DATE = $(shell date -u -d "@$(SOURCE_DATE_EPOCH)" '+%d %b %Y' 2>/dev/null || date -u -r "$(SOURCE_DATE_EPOCH)" '+%d %b %Y')
 HUB_VERSION = $(shell bin/hub version | tail -1)
 FLAGS_ALL = $(shell go version | grep -q 'go1.[89]' || echo 'all=')
-export LDFLAGS := -extldflags='$(LDFLAGS)'
-export GCFLAGS := $(FLAGS_ALL)-trimpath='$(PWD)'
-export ASMFLAGS := $(FLAGS_ALL)-trimpath='$(PWD)'
+export LDFLAGS := -extldflags '$(LDFLAGS)'
```

Comment:
Let's remove the quotes instead and put back the `=` and have people write to us if they experience problems with values that include spaces.

Reply:
Assuming the rationale is because you're not sure either, how the arguments are meant to be passed in such cases?

Done.

Reply:
Oh I thought my addition of the quotes was the problem? And that removing them will solve it. But you're saying that doesn't completely solve it? I'm not sure about the proper solution, since I don't use these compiler flags at all, so I'll defer to your judgement.